sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -272,23 +273,20 @@ bool test_main(void) {
       CHECK(dump.double_fault == kDifToggleEnabled,
             "CPU Info dump doesn't show a double fault has happened.");
 
-      // The current behaviour after a double fault is to capture, in the CPU
-      // info dump, the interrupt vector below the one which was taken to jump
-      // to the exception handler as the current PC and the start of the
-      // exception handler as the next PC. This feels wrong. However, with a
-      // lack of a clear definition of what these values should contain, the
-      // test enforces this behaviour so that regressions can be caught.
-      uint32_t curr_pc = (uint32_t)&_ottf_interrupt_vector + 4;
-      uint32_t next_pc = (uint32_t)&handler_exception;
```

Comment:
The `handle_exception` symbol was only used here, so can be removed from the rest of the file.

Reply:
removed!